_dashboards/csp/csp-dynamic-configuration.md

```diff
@@ -0,0 +1,50 @@
+---
+layout: default
+title: Content Security Policy (CSP) rules dynamic configuration
```

Comment:
Should this match H1? Or "Dynamic configuration of CSP rules"?

Reply:
Yes. I missed that change.